docs/design/data-loading.mdx

```diff
@@ -816,6 +817,7 @@ Initializes the OracleDB data loading client.
     -   `host (str)`: Host address for database.
     -   `port (int)`: Port on which the database is running Defaults to 3306.
     -   `service (str)`: OracleDB service
+    -   `mode (str)`: switch between OracleDB mode of thin or thick
```

Comment:
Can you wrap the possible values with backticks: Switch between OracleDB mode of `thin` or `thick`